ifrs17/Import/Importers.ipynb

```diff
@@ -373,32 +419,7 @@
                 "\n    var args = GetArgsFromMain(dataSet);",
                 "\n    if(ApplicationMessage.HasErrors()) return null;",
```

Comment:
Why this check here? This should never be null at this point. If it is, we miss a validation earlier in the process.

Reply:
We have validations in the GetArgsFromMain method. The check in line 374 serves to check whether those validations failed, and in case they did, it has to return.

Reply:
I understand, but shouldn't we do this inside the GetArgsFromMain, imposing that this method always gives valid args?
Also, if you use GetArgsFromMain in other places, you would have to implement similar checks. Thus IMO the check must be within the method, you see?

Reply:
I am talking on like 375, not 374.

Reply:
GetArgsFromMain can return an args with ReportingNode null. This happens for instance in the YieldCurve importer, for which the main tab has only Year Mont Scenario. Conversely, GetArgsFromMain has other kind of checks, e.g. main tab exists, has entry, etc. Therefore, since the check in line 375 is specific for the GetArgsAndCommitPartitions method, because the intention is also to commit the partition and hence the ReportingNode must be provided, this line was placed (not by this PR) outside of the method GetArgsFromMain.

Reply:
Let's discuss and have a dedicated ticket.